example/pubspec.lock

```diff
@@ -59,7 +73,7 @@ packages:
       name: vector_math
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "2.1.0"
+    version: "2.1.1"
 sdks:
-  dart: ">=2.12.0-259.9.beta <3.0.0"
-  flutter: ">=1.12.13+hotfix.6"
+  dart: ">=2.15.0 <3.0.0"
+  flutter: ">=2.8.1"
```

Comment:
For plugins the recommended approach is not to commit the `pubspec.lock` file into source control. Best to add it to the `.gitignore` file and remove it from the git repository.